src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_RENTAL_COST = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const LONG_TERM_DURATION = 7;
+  const SHORT_TERM_DURATION = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const TOTAL_PRICE = DAY_RENTAL_COST * days;
```

Comment:
The `TOTAL_PRICE` should be calculated after applying the discount. Currently, it calculates the full price without considering any discounts. Consider moving the `TOTAL_PRICE` calculation inside the conditional blocks where discounts are applied.